knx/dpt/types_20.go

```diff
@@ -0,0 +1,33 @@
+package dpt
+
+// DPT_20102 represents DPT 20.102 / HVAC Mode.
+type DPT_20102 uint8
+
+func (d DPT_20102) Pack() []byte {
+	return packU8(uint8(d))
+}
+
+func (d *DPT_20102) Unpack(data []byte) error {
+	return unpackU8(data, (*uint8)(d))
+}
+
+func (d DPT_20102) Unit() string {
+	return ""
+}
+
+func (d DPT_20102) String() string {
+	switch d {
+	case 0:
+		return "Auto"
+	case 1:
+		return "Comfort"
+	case 2:
+		return "Standby"
+	case 3:
+		return "Economy"
+	case 4:
+		return "Building Protection"
+	default:
+		return ""
```

Comment:
This is actually "reserved".

Reply:
In the KNX master file of my test project this was not named, but ETS gave me an empty string.
What source du you usually use?

Reply:
It is defined in the KNX standard.

![image](https://user-images.githubusercontent.com/2806669/187203792-854c7d15-60f3-4bd4-b802-1ca78bd0226e.jpeg)

Reply:
True, you find this in the types definition a lot in various versions. There is for example `reserved`, `reserved, unused`, `not used, reserved`, but in my understanding this is all referring to an unused value (-range) and not the value, or translation of these. Implementations, like the ETS, does not take them into account as well.
I would suggest to not translate this, but in the end it's your project, if you want to have it in I will change it. Just leave me your thoughts on this. :)

Reply:
The problem is that the empty string doesn't mean anything. Somebody encountering that who is not familiar with this discussion would assume something is wrong. Hence there must be some meaning conveyed. I am not too fixed on "reserved" - for all I care it could be "unspecified", "unknown" or similar. However, the empty string is simply not an acceptable choice.

Reply:
I see your point, sounds good!
Maybe "unknown" is the best bet to go for when using it throughout the project.

Not said so far - thanks for all your work here, really appreciate it! :)

Reply:
I would prefer "reserved", since it is what the spec. says. "unknown" may imply it is unknown on the receiver side, but maybe not on the sender side, but we know that the value is reserved and currently not used.

Reply:
Changed